File: 02 Digits and Numbers - Foundation Programs/program-08.cpp
// Write a program that takes a 3-digit number as input and swaps its first and last digits, then displays the
// result (e.g., 456 becomes 654).


#include<iostream>
using namespace std;
int main()
{
    int number,rev_digit;
    cout<<"Enter 3-digit number : ";
    cin>>number;
    
    rev_digit=(number%10)*100;  //123 as input number we got 3*100 = 300
    rev_digit=rev_digit+((number/10)%10)*10;
    rev_digit=rev_digit+(number/100);

    cout<<"reverse order : "<<rev_digit;

    return 0;
}

File: archit_vishwakarma/Level-1/03-Digits-And-Numbers/program-08.py
n = input("\nEnter a number : ")

try:
    n = int(n)
    if n < 1000 and n > 99:

        a = ((n%10)*100) + (n%100)
        b = ((a//10)*10) + (n//100)
        
        print(f"after swaping first and last digit : {b}\n")

    else:
        print("Enter a Three Digit Number")
except ValueError:
    print("enter a valid number")


